src/test/java/rv/atqc/gttesting/driverfactory/Driver.java

```diff
@@ -0,0 +1,24 @@
+package rv.atqc.gttesting.driverfactory;
+
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public abstract class Driver {
+    public RemoteWebDriver init( DesiredCapabilities desiredCapabilities){
+        int port = 32773;
```

Comment:
If we want to change port, for working with other Selenium Grid - we have to change this port in all classes. This good for presentation but not for work.